src/main/modules/api/cos-api.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosInstance } from 'axios';
+import { LoggerInstance } from 'winston';
+
+export class COSApi {
+  constructor(
+    private readonly axios: AxiosInstance,
+    private readonly logger: LoggerInstance
+  ) { }
+
+  public createCase(data: any): Promise<boolean> {
```

Comment:
Maybe something better than any?

Reply:
oh, initially i wasnt sure what the payload would be but ill change it after talking to Sarah.